test/1.4/lib/T_largearray.dml

```diff
@@ -12,8 +12,8 @@ bank b {
     register a[i < 10000] @ 50000 + (4 * i) is read_only;
     register b[i < 10000] @ 90000 + (4 * i) is read_only;
 
-    register c[i < 10000] @ 150000 + (8 * i) is read_only;
-    register d[i < 10000] @ 150004 + (8 * i) is read_only;
+    register c[i < 100000] @ 1500000 + (8 * i) is read_only;
+    register d[i < 100000] @ 1500004 + (8 * i) is read_only;
```

Comment:
The penalty for the 100000 size is around 0.1 s. The biggest cost is now is the 3M register dispatches triggered by read(), takes around 1s. Perhaps we should reduce that loop by an order of magnitude.

I want the test's running time to be close to annoying, so that a significant regression will be noticeable when running the test suite manually.